### DISCUSSION TITLE
Newsletter

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#14] [Idee/Proposte]
Spulciando il sito ho notato la sezione "[Newsletter](http://opendatasicilia.it/newsletter/)". È ancora in vita? Dovremmo mantenerla nel nuovo sito?

**Comment by cirospat** [comment]
Secondo me va rivitalizzata la newsletter, considerato che le attività di ODS continuano con costanza nel tempo attraverso le varie iniziative dei suoi membri.

Penso..
Se sto lavorando ad una specifica iniziativa, devo documentarla da qualche parte (repo GitHub ODS newsletter) così che a 15 giorni (ad es.) è già compilato il contenuto della newsletter che deve partire (anche con automatismo).
Sto pensando....

**Comment by dennisangemi** [comment]
@cirospat idea bellissima, forse però la newsletter necessità di un lavoro alle spalle non indifferente (mi viene in mente tutto ciò che fa @aborruso per quella di onData). Che ne dite invece di programmare un invio periodico (e.g. una volta al mese) con una sorta di riepilogo di tutti i nuovi post pubblicati sul nostro sito? Insomma, avremmo una newsletter con tutte le novità di ODS che potrebbe essere automatizzata (non so se è possibile realizzare una cosa del genere).
